server/src/main/java/com/exacaster/lighter/rest/SessionParameters.java

```diff
@@ -0,0 +1,25 @@
+package com.exacaster.lighter.rest;
+
+import com.exacaster.lighter.application.SubmitParams;
+import io.micronaut.core.annotation.Introspected;
+
+import java.util.List;
+import java.util.Map;
+
+@Introspected
+public class SessionParameters extends SubmitParams {
+    private final Boolean permanent;
+
+    public SessionParameters(Boolean permanent, String name, String file, String master, String mainClass, Integer numExecutors, Integer executorCores,
```

Comment:
I think we need to add `@Nullable` annotations here, as we do on `SubmitParams`, otherwise Micronaut validation fails (I'm not completely sure about it)

Reply:
`@MapFormat(transformation = FLAT, keyFormat = RAW)` should also be added

Reply:
done